packages/jetbrains-toolbox-app/debian/postrm

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+rm -f /opt/jetbrains-toolbox-app.tar.gz # Remove the empty file, if the download failed.
+rm -rf /opt/jetbrains-toolbox-app
+rm -rf ~/.local/share/JetBrains # It installs itself on first boot in this folder.
```

Comment:
According to https://toolbox-support.jetbrains.com/hc/en-us/articles/115000978804-Where-are-the-IDEs-located-on-my-hard-drive- this is also where the Toolbox App puts the IDEs. Do we really want to remove all IDEs on toolbox installation?

Reply:
I assumed, that, when you you want to remove the toolbox, you also want to remove every IDE.

But that could be the wrong way.
I switch it to a Hint, that the User **should** remove this folder by hand, if he wants to remove the IDE's.